canonical-websites/www.ubuntu.com
# Installed the packaged version of the global-nav

## Done
Removed the local global nav assets and installed the packaged global-nav.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the global navigation loads and works correctly
- Check on small screens
- Check it doesn't break any links in and around other pages
- Check there are no JS errors in the console.

## Issue / Card
Fixes #4077
